app/src/main/res/values/strings.xml

```diff
@@ -155,6 +155,7 @@
     <string name="tx_status_type_creation">Safe created</string>
 
     <string name="tx_status_status_awaiting_confirmations">\u2022 Awaiting confirmations</string>
+    <string name="tx_status_status_awaiting_your_confirmation">\u2022 Awaiting your confirmation</string>
```

Comment:
Do we need the duplication of these Strings? I probably introduced it but I don't remember whether I had a good reason :-)

Reply:
no, we don't.  I'll clean it up by leaving only status labels